src/functions-and-arrays.js

```diff
@@ -78,8 +117,12 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
-
+function howManyTimes(wordsCount, word) {
+  if (wordsCount.includes(word)) {
+    return word
```

Comment:
Como ele pede o número de vezes que a palavra aparece, não dá pra usar includes. Porque esse método retorna somente true/false caso tenha a palavra ou não. 
Um jeito de fazer seria:
function howManyTimes(wordsCount, word) {
  let count = 0;
  for (let element of wordsCount) {
    if (element === word) count += 1;
  }
  return count;
}